sdk/tests/types/address/bech32.rs

```diff
@@ -40,7 +37,10 @@ fn ctors() {
 fn hrp_from_str() {
     Hrp::from_str("rms").unwrap();
 
-    assert!(matches!(Hrp::from_str("中國"), Err(Error::InvalidBech32Hrp(_))));
+    assert!(matches!(
+        Hrp::from_str("中國"),
+        Err(bech32::primitives::hrp::Error::NonAsciiChar(_))
```

Comment:
Shouldn't we wrap this error?

Reply:
why?

Reply:
Seems a bit weird to me to leak this that much, we usually don't 🤷🏻‍♂️

Reply:
What error would you want this to be part of? Address?

Reply:
Yeah sounds fitting to me